### DISCUSSION TITLE
Colouring text in node cores

### DISCUSSION THREAD

**Discussion by GusGF** [freeplane/freeplane#553] [Help] [answered]
When I'm editing/decorating text i.e. like changing the colour of the highlighted text during an edit of a node core how do I repeat the operation without having to repeatedly bring up the colour palette. You will see an example of blue text here: 

![image](https://user-images.githubusercontent.com/21323227/183889055-0e9eee11-919f-4582-9f92-f838b03c844f.png)

**Comment by euu2021** [comment, marked as answer]
there are hotkeys for some colors. Right click to see the hotkeys:

![image](https://user-images.githubusercontent.com/77707706/183903829-acd74eea-4c2e-4b3b-936c-a765009e6fe9.png)

**Reply by GusGF** [reply]
Brilliant thank you :)
